Unable to build

Hi!

First of all, thanks for this awesome module!

Second:

```
/Users/avi/Developer/sensei/3rdParty/addons/ofxTimeMeasurements/src/ofxTimeMeasurements.cpp:1682:2: error:
      use of undeclared identifier 'ofRotateDeg'; did you mean 'ofRotate'?
        ofRotateDeg(r, 0, 0, 1);
        ^~~~~~~~~~~
        ofRotate
/Users/avi/Developer/sensei/3rdParty/of/src/openframeworks/graphics/ofGraphics.h:953:6: note:
      'ofRotate' declared here
void ofRotate(float degrees, float vecX, float vecY, float vecZ);
     ^
1 error generated.
make[2]: *** [CMakeFiles/sensei.dir/3rdParty/addons/ofxTimeMeasurements/src/ofxTimeMeasurements.cpp.o] Error 1
make[1]: *** [CMakeFiles/sensei.dir/all] Error 2
make: *** [all] Error 2
```

The problem was trivially fixed by changing "ofRotateDeg" to "ofRotate".

--- a/src/ofxTimeMeasurements.cpp
+++ b/src/ofxTimeMeasurements.cpp
@@ -1700,7 +1700,7 @@ void ofxTimeMeasurements::drawSmoothFpsClock(float x, float y, float radius){
 	static float r = 0;
 	ofPushMatrix();
 	ofTranslate(x, y);
-	ofRotateDeg(r, 0, 0, 1);
+	ofRotate(r, 0, 0, 1);
 	float s = radius * 0.05;
 	ofDrawRectangle(-s * 0.5f, - 0.5f * s, radius, s);
 	ofPopMatrix();

